bucket/waifu2x-extension-gui.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "3.92.01",
+    "description": "Video, Image and GIF upscale/enlarge(Super-Resolution) and Video frame interpolation. Achieved with Waifu2x, Real-ESRGAN, Real-CUGAN, SRMD, RealSR, Anime4K, RIFE, CAIN, DAIN, and ACNet.",
+    "homepage": "https://github.com/AaronFeng753/Waifu2x-Extension-GUI",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://github.com/AaronFeng753/Waifu2x-Extension-GUI/blob/master/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/AaronFeng753/Waifu2x-Extension-GUI/releases/download/v3.92.01/Waifu2x-Extension-GUI-v3.92.01-Win64.7z",
+            "hash": "e64e47a0957f6ce3337240d5b17c2db42c7ad6e09665b1e6ee4bbc1adc9303fb",
+            "extract_dir": "waifu2x-extension-gui"
+        }
+    },
+    "shortcuts": [
+        [
+            "Waifu2x-Extension-GUI.exe",
+            "Waifu2x-Extension-GUI"
+        ]
+    ],
+    "bin": "Waifu2x-Extension-GUI.exe",
+    "checkver": {
+        "github": "https://github.com/AaronFeng753/Waifu2x-Extension-GUI"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/AaronFeng753/Waifu2x-Extension-GUI/releases/download/v$version/Waifu2x-Extension-GUI-v$version-Win64.7z",
+                "hash": {
+                    "url": "https://github.com/AaronFeng753/Waifu2x-Extension-GUI/releases/tag/v$version",
+                    "mode": "extract",
+                    "regex": "Waifu2x-Extension-GUI-v$version-Win64.7z\\n$sha256"

```

```suggestion
                    "regex": "$basename\\n$sha256"
```
`extract` mode is default mode, so `"mode": "extract"` can be removed.

By the way, we have to **escape** `.` to `\\.` as well. Otherwise it will be **"match any character"**.